app/helpers/application_helper.rb

```diff
@@ -20,7 +20,7 @@ def format_datetime_timespec(time, format)
   end
 
   def format_currency(amount)
-    class_name = amount < 0 ? 'negative_amout' : 'positive_amount'
+    class_name = amount < 0 ? 'negative_amout' : 'positive_amount' if amount.is_a? Numeric
     content_tag :span, number_to_currency(amount), class: class_name
```

Comment:
I think this works, but is a little strange: if amount is not numeric, then `class_name` is actually undefined.

Reply:
should we set class_name to a default value then?

Reply:
Is it about `amount` being `nil`? In that case, we can just omit the `span` entirely and return `nil` directly.